internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -91,6 +93,15 @@ type (
 		startedAt          time.Time
 	}
 
+	userNewPayload struct {
+		StreamName   	   string   `json:"streamName"`
```

Comment:
If we're only sending 1 information about the stream I believe it should be `streamId` instead.

We could also send "everything" like a full `stream` object, I only think it's complicated cause we'd have to filter any admin fields we might have received about it. So I think just the ID is good, and they can call the API for the full object if desired.

Reply:
In the USER_NEW payload `streamName` is not the the stream ID but actually the playback ID. I think `StreamID` here would make it confusing, as well as `PlaybackID` so I kept the Mist key name, wdyt?

Reply:
I think webhook payloads are just like API endpoints. They are part of the public interface of the API. As such, they should use a consistent language between resources.

On this case, this StreamName is not something that makes sense for our current API abstractions. It would seem like it's the stream.name, but it's something else. For users of the API that something else also has an existing name, playbackId. So that's what I think we should would call it, even this didn't have to change for consistency with the other API webhooks anyway.

So you should just follow the schema that I linked in the other comment. You would just have a stream object here with all the fields of the stream, among some other default metadata fields.

There is also the payload field which is the freeform one, each webhook has its own. I think it would make sense to setup that webhook using the concepts of our current API, before a new one is redesigned. If you prefer to use mist terms tho, at least they will be isolated inside this payload field so it's likely less confusing.

Reply:
I gotta give this out today, so I'm going to strip the hook down to just this:

```
{
  "payload": {
    "requestUrl": "http://playback.livepeer.systems:8081/hls/d6b8nhc03ofgwx6v/index.m3u8?stream=d6b8nhc03ofgwx6v&proof=abc123"
  }
}
```

That'll presumably stay stable and contains all the information that they need to implement token-gating. We can add other fields and make more consistent based on what we want

Reply:
Er, I guess we need timestamp as well for HMAC confirmation?